sagemaker-debugger/tensorflow2/tensorflow2_keras_to_estimator/tf2-keras-model-to-estimator.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>We will train a TensorFlow Keras model in this notebook with Amazon Sagemaker Debugger, and monitor the training jobs with the Debugger built-in rules. The training job will be run on a pre-built <a href="https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-images.html" target="_blank">AWS Deep Learning Container</a> with Tensorflow 2.1.0 and smdebug 0.9.0 installed. </p><p> </p><p>Execute the following code cell for a one-time smdebug setup to get your notebook kernel ready for a full experience of using the Debugger features. This smdebug library provides tools to perform interactive analysis throughout the notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done in latest</p>